ci-operator/step-registry/cucushift/hypershift-extended/enable-qe/pull-secret/cucushift-hypershift-extended-enable-qe-pull-secret-ref.yaml

```diff
@@ -3,7 +3,7 @@ ref:
   from: upi-installer
   cli: latest
   grace_period: 10m
-  timeout: 55m0s
+  timeout: 60m0s
```

Comment:
why from 55 to 60? 55 is timeout?

Reply:
yes, The maximum execution time of this step is 60m